docs/api/rest_api/rest_api_usage/rest_api_usage.md

```diff
@@ -42,7 +42,7 @@ Accept: application/vnd.ibexa.api.RelationList+xml
 Resources that accept a reference to another resource expect the reference to be given as a REST URI, not a single ID.
 For example, the URI requesting a list of user groups assigned to the role with ID 1 is:
 
-```
+```http
```

Comment:
That a request, not a response, and `http` highlight seems to work only on responses.

```suggestion
```
```

Reply:
Work on request with full syntax. I added ` HTTP/1.1` to fix it.